packages/styles/scss/components/modal/_modal.scss

```diff
@@ -25,15 +25,10 @@
   .#{$prefix}--modal {
     position: fixed;
     z-index: z('modal');
-    top: 0;
-    left: 0;
     display: flex;
-    width: 100vw;
-    height: 100vh;
```

Comment:
I worry that other implementations of the modal in angular, svelte, etc. would be adversely affected by this change. @tw15egan do you think this would be a breaking change? We usually don't for styles, but I'm not totally sure in this case.